pynest/nest/tests/test_rate_neuron_communication.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# test_rate_neuron_communication.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# This script tests the rate_neuron in NEST.
+
+import nest
+import unittest
+import math
+import numpy as np
+
+
+def H(x):
+    return 0.5 * (np.sign(x) + 1.)
+
+
+@nest.check_stack
+class RateNeuronCommunicationTestCase(unittest.TestCase):
+    """Check rate_neuron"""
```

Comment:
You could extend this documentation by giving some more details, such as:
This test checks the correct function of rate neuron with input noise and three transfer functions. It also checks whether delay and weight of the rate connections work correctly.